### ISSUE TITLE
Function to convert all emojis from a string to html code

### ISSUE THREAD

**Issue by marcelcaferati** [felldo/JEmoji#5]
### What feature are you proposing?

I dont know if this is a very specific use case. 

Untill now i was using VRDUMOND, but since that died several years ago and it no longer support newer emojis, i migrate to this one.

I have this need to take a regular string (comment from user) and parse all the emojis to htmldecimal, i achieved this using the code bellow after a quick read to the readme. So just leaving this here in case you guys want to do something with it, i am not sure this code is really optimized based on what else is available in this lib since I didn't spend long looking ehhe
```java
public static String convertEmojisToHtmlDecimal(String input) {
        StringBuilder output = new StringBuilder();
        for (int i = 0; i < input.length(); i++) {
            int codePoint = input.codePointAt(i);
            if (JEmoji.isEmoji(new String(Character.toChars(codePoint)))) {
                output.append("&#").append(codePoint).append(";");
                if (Character.isSupplementaryCodePoint(codePoint)) {
                    i++;
                }
            } else {
                output.append(input.charAt(i));
            }
        }
        return output.toString();
    }
```

**Comment by felldo** [comment]
Not sure what code this is, but this is not how you use `JEmoji`. Most of the methods are available in the `EmojiManager` class and the `JEmoji` class you are using doesn't exist.

If you are looking for a better way to write your code your can use 
```java
EmojiManager.extractEmojis(input).stream().map(Emoji::getHtmlDecimalCode).collect(Collectors.joining());
``` 
which should do the same as your code. 
I don't think the library would benefit of it that much, because it can be achieved quiet easy with the code above.

**Comment by marcelcaferati** [comment]
Sorry I kinda screwed up with the JEmoji class name hehe i was testing things around, I ended up achieving the same with EmojiManager as you mentioned.

I see the code provided looks much better! Looks like i went the long way and missed that extractEmoji method

Thank you

**Comment by felldo** [comment]
No problem :)
Btw. I used the wrong method. It's `extractEmojisInOrder` which returns you a list of all emojis (even duplicated ones) while `extractEmojis` does not.

**Comment by marcelcaferati** [comment]
Yea, i was testing out your code, it doenst not achieve what I need.

I need to have Input and string like "Hello! 😑" and have it return to me the same string "Hello! emojiHtmlCode"

The code you provided actually only returns all the emojis and replaces them with the code, discarding the rest of the string.

**Comment by felldo** [comment]
In this case you have found a good new method which can be added.
Something like `replaceAllEmojis(String input, Function<Emoji, String> function)`. This should allow you to do any custom replacement function you want. 
I'm going to add something later this day :)

**Comment by felldo** [comment]
I've released a new version `1.2.0` where you can now use the code below to replace the emojis with their html decimal code
```java
EmojiManager.replaceAllEmojis("Hello 😀 World 👍", Emoji::getHtmlDecimalCode);
```
